### PR TITLE
Feature/calculator body

### PR DESCRIPTION
**Pull request by Tchilo**
- Add component folder 
- Add calculator file 
- Make calculator body
- Add styles to the calculator

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,26 @@
...
+//           rel="noopener noreferrer"
+//         >
+//           Learn React
+//         </a>
+//       </header>
+//     </div>
+//   );
+// }
 
-function App() {
```
Comment (GzimAsani) [inline src/App.js:4]: - [x] All the boilerplate that got created by the CRA (texts, images, and styles) should be deleted.

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+function Calculator() {
```
Comment (GzimAsani) [inline src/components/calculator.js]: - [x] The `Calculator.js` component should be class-based component

File: src/App.js [hunk 3]
```diff
@@ -1,29 +1,26 @@
...
+//           href="https://reactjs.org"
+//           target="_blank"
+//           rel="noopener noreferrer"
+//         >
+//           Learn React
+//         </a>
+//       </header>
+//     </div>
+//   );
+// }
```
Comment (GzimAsani) [inline src/App.js]: - [x] The app should be a functional and class-based component as this should be the only component that will be sent to the index.js file, all the other files should be sent here in the app.js file

File: src/index.js [hunk 4]
```diff
@@ -1,17 +1,15 @@
...
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/calculator';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```
Comment (GzimAsani) [inline src/index.js]: - [x] Please remove the calculator component. The app component should be displayed here and all the other components should be sent to the app.js file

File: src/index.css [hunk 5]
```diff
@@ -1,27 +1,38 @@
 body {
```
Comment (GzimAsani) [inline src/index.css:1]: - [x] Your calculator should match the given design, please follow more closely the design colors and position

Given design:
Screenshot from 2021-10-27 19-45-23

Yours:
Screenshot from 2021-10-27 19-44-59

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @Tchilo                                  
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Gitflow is used :clap:
- Good PR description :clap: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 5 above)
Comment (Tchilo) [reply src/index.css:1]: changes made

**Review by AshakaE — APPROVED** [review]
### Hi @Tchilo  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**